src/proxy/ClientTransfer.php

```diff
@@ -37,6 +37,9 @@ public function start()
                 }
             }
         }
+
+        $this->build->command->outputInfo('Using local database ' . $this->build->db->createCommand('SELECT DATABASE()')->queryScalar());
```

Comment:
If this won't work for any DMBS system, we'll need to wrapp your schema instance check.